### PR TITLE
analysis: Prefer using glib::prelude::*

### LINKED ISSUE #1401: Generates unused imports
GIR seems to generate unused import statements.

https://gitlab.gnome.org/World/Rust/tracker-rs/

    $ cargo build

```
warning: unused import: `glib::StaticType`
  --> src/auto/endpoint_dbus.rs:11:5
   |
11 | use glib::StaticType;
   |     ^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

warning: unused import: `glib::StaticType`
  --> src/auto/endpoint_http.rs:17:5
   |
17 | use glib::StaticType;
   |     ^^^^^^^^^^^^^^^^

warning: unused import: `glib::object::Cast`
 --> src/auto/notifier.rs:7:5
  |
7 | use glib::object::Cast;
  |     ^^^^^^^^^^^^^^^^^^

warning: unused import: `glib::StaticType`
  --> src/auto/notifier.rs:10:5
   |
10 | use glib::StaticType;
   |     ^^^^^^^^^^^^^^^^

warning: unused import: `glib::object::Cast`
 --> src/auto/resource.rs:7:5
  |
7 | use glib::object::Cast;
  |     ^^^^^^^^^^^^^^^^^^

warning: unused import: `glib::StaticType`
  --> src/auto/resource.rs:12:5
   |
12 | use glib::StaticType;
   |     ^^^^^^^^^^^^^^^^

warning: unused import: `glib::object::ObjectType`
 --> src/auto/notifier.rs:8:5
  |
8 | use glib::object::ObjectType as ObjectType_;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `glib::object::ObjectType`
 --> src/auto/endpoint_dbus.rs:9:5
  |
9 | use glib::object::ObjectType as ObjectType_;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^

warning: `tracker-rs` (lib) generated 8 warnings
    Finished dev [unoptimized + debuginfo] target(s) in 1.60s
```

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Reduce the amount of generated code and drops a bit of unneeded complexity

Fixes #1401

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
When applied to gtk4-rs `581 insertions(+), 1574 deletions(-)`

**Comment by bilelmoussaoui** [comment]
I am also looking at grouping those kind of imports, that would simplify things a bit further
```
use crate::Accessible;
use crate::AccessibleRole;
use crate::Align;
use crate::Buildable;
use crate::ConstraintTarget;
use crate::Editable;
use crate::LayoutManager;
use crate::Overflow;
use crate::Widget;
```

File: src/analysis/bounds.rs [hunk 1]
```diff
@@ -275,7 +275,7 @@ impl Bounds {
         for used in &self.used {
             match used.bound_type {
                 NoWrapper => (),
-                IsA(_) => imports.add("glib::object::IsA"),
+                IsA(_) => imports.add("glib::prelude::*"),
```
Comment (sdroege) [inline src/analysis/bounds.rs:278]: This one should not really be needed as `crate::prelude::*` should re-export it or not?

**Review by sdroege — APPROVED** [review]
Good for me otherwise

**Comment by sdroege** [comment]
> I am also looking at grouping those kind of imports, that would simplify things a bit further
> 
> ```
> use crate::Accessible;
> use crate::AccessibleRole;
> use crate::Align;
> use crate::Buildable;
> use crate::ConstraintTarget;
> use crate::Editable;
> use crate::LayoutManager;
> use crate::Overflow;
> use crate::Widget;
> ```

You could post-process them to make them all `crate::{....}`

File: src/analysis/bounds.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply src/analysis/bounds.rs:278]: We don't add crate::prelude::* anywhere yet

**Comment by bilelmoussaoui** [comment]
> > I am also looking at grouping those kind of imports, that would simplify things a bit further
> > ```
> > use crate::Accessible;
> > use crate::AccessibleRole;
> > use crate::Align;
> > use crate::Buildable;
> > use crate::ConstraintTarget;
> > use crate::Editable;
> > use crate::LayoutManager;
> > use crate::Overflow;
> > use crate::Widget;
> > ```
> 
> You could post-process them to make them all `crate::{....}`

yes that is what i am planning to do, in a different PR though :)

File: src/analysis/bounds.rs (hunk 1 above)
Comment (sdroege) [reply src/analysis/bounds.rs:278]: Ok so this can get merged then for now, and other things are for other PRs? :)

Comment (bilelmoussaoui) [reply src/analysis/bounds.rs:278]: that would be my preference

**Pull request merged**